pkg/driver/memory/device_manager_memory.go

```diff
@@ -252,6 +262,7 @@ func (d *DeviceManager) DeviceRegister(cert, onboard *x509.Certificate, serial s
 		Metrics: &ByteSlice{
 			maxSize: d.maxMetricSize,
 		},
+		AppLogs: map[uuid.UUID]common.BigData{},
```

Comment:
Same question here. And where does the member of the `AppLogs` get instantiated?